src/request/index.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+interface ISignupBody {
+  email: string;
+  nickname: string;
+  image: File;
+}
+
+interface IRequestStampBody {
+  omakaseId: number;
+  receiptIssuaranceData: string;
+  receiptImage: File;
+}
+
+const instance = axios.create({ baseURL: process.env.API_ENDPOINT });
+
+export const setTokenOnHeader = (token: string) => {
+  instance.defaults.headers.common['Authorization'] = token;
```

Comment:
```suggestion
  instance.defaults.headers.common['Authorization'] = `Bearer ${token}`;
```
저희 명세에 따르면 `Bearer token`이라 `Bearer` 추가되어야 할 거 같슴둥

Reply:
추가했습니다 감사합니다~